### PR TITLE
Add C# meeting

### PR DESCRIPTION
**Pull request by jsturtevant**
Adding c# meeting details

### CODE REVIEW

**Comment by esoterra** [comment]
Is `Csharp` the conventional spelling? we use hyphens in directory names so `C-Sharp` would be allowed.

File: SIG-Guest-Languages/Csharp/README.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+A meeting for [the Special Interest Group (SIG) on Guest Languages for Wasm, focusing on C#](https://github.com/bytecodealliance/SIG-Guest-Languages/pull/7).
+
+## Time and Location
+
+**When**: every other Tuesday at 8:15am PST (5:15pm CET)
+**Where**: Google Meet (https://meet.google.com/qsi-giid-vib)
+
+## Attending
+
+Join the following google group: https://groups.google.com/g/ba-sig-guest-languages/c/2ovNM27dp7U, invite in past emails.
```
Comment (esoterra) [inline SIG-Guest-Languages/Csharp/README.md:12]:
```suggestion
Join [the SIG Guest Languages google group](https://groups.google.com/g/ba-sig-guest-languages), the meeting invite is in past [emails](https://groups.google.com/g/ba-sig-guest-languages/c/2ovNM27dp7U).
```

**Comment by jsturtevant** [comment]
> Is `Csharp` the conventional spelling? we use hyphens in directory names so `C-Sharp` would be allowed.

Not 100% sure, but that is how it is referenced in repositorys like https://github.com/dotnet/csharplang when not using C#

**Comment by esoterra** [comment]
Fair enough, what do you think of capitalizing it as "CSharp" though?
Something about the "C" being capital and the "sharp" being lowercase looks strange to me.

Not a blocking issue though, especially because this is a pretty easy rename.

**Review by esoterra — APPROVED** [review]

**Comment by jsturtevant** [comment]
I actually had it all lower case first then uppercased it to match the rest of the repo.  🤷

**Review by yowl — APPROVED** [review]

**Comment by tomkerkhove** [comment]
Do we have Google calendars to subscribe to?

**Comment by yowl** [comment]
@tomkerkhove Not as far as I know, but if you DM me in the zulip chat with your email, I will forward the invite to you.

**Comment by silesmo** [comment]
> Do we have Google calendars to subscribe to?

If you join the SIG-Guest Lanaguages google group, then you should get an invite: https://groups.google.com/g/ba-sig-guest-languages @tomkerkhove

**Pull request merged**